src/main/java/ca/corefacility/bioinformatics/irida/repositories/filesystem/IridaFileStorageLocalUtilityImpl.java

```diff
@@ -260,4 +285,11 @@ public boolean checkWriteAccess(Path baseDirectory) {
 		return true;
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public boolean isStorageTypeLocal() {
+		return storageType.equals(StorageType.LOCAL);
+	}
```

Comment:
These implementations seem odd in all the filestorageutility classes.  Can they just return `true` or `false` as needed?

Reply:
Yup that works! Updated in 50d4f4fd7d2a884fc1dc8abc61771bc106536aad